<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>511</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_a_Bitmap">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_a_Bitmap"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionChange_Color"/>
    <addaction name="actionDulplicate"/>
    <addaction name="actionGradual_disappearence"/>
   </widget>
   <addaction name="menuOpen_a_Bitmap"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_a_Bitmap">
   <property name="text">
    <string>Open a Bitmap</string>
   </property>
  </action>
  <action name="actionChange_Color">
   <property name="text">
    <string>Change Color</string>
   </property>
  </action>
  <action name="actionDulplicate">
   <property name="text">
    <string>Dulplicate</string>
   </property>
  </action>
  <action name="actionGradual_disappearence">
   <property name="text">
    <string>Gradual disappearence</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
